lib/inferno/dsl/runnable.rb

```diff
@@ -200,8 +200,16 @@ def description(new_description = nil)
       # @return [void]
       # @example
       #   input :patient_id, :bearer_token
-      def input(*input_definitions)
-        inputs.concat(input_definitions)
+      def input(name, *other_names, **input_definition)
+        if other_names.present?
+          [name, *other_names].each do |input_name|
+            inputs.push({ name: input_name, title: nil, description: nil, type: nil })
+          end
+        elsif input_definition.present?
```

Comment:
You can just have an else here. `**` arguments default to `{}` rather than `nil`.

Reply:
Fixed